mmv1/templates/terraform/post_create/compute_region_backend_service_security_policy.go.erb

```diff
@@ -0,0 +1,27 @@
+<% unless version == 'ga' -%>
+// security_policy isn't set by Create / Update
+if o, n := d.GetChange("security_policy"); o.(string) != n.(string) {
+  pol, err := tpgresource.ParseSecurityPolicyRegionalFieldValue(n.(string), d, config)
+  if err != nil {
+    return errwrap.Wrapf("Error parsing Region Backend Service security policy: {{err}}", err)
+  }
+
+  region, err := tpgresource.GetRegion(d, config)
+  if err != nil {
+    return err
+  }
+
+  spr := emptySecurityPolicyReference()
+  spr.SecurityPolicy = pol.RelativeLink()
+
+  op, err := config.NewComputeClient(userAgent).RegionBackendServices.SetSecurityPolicy(project, region, obj["name"].(string), spr).Do()
```

Comment:
This resource works via the generated HTTP calls, I think that introducing a requirement on the client library here is going to be an issue. Can we make this call directly to the API? Maybe using some of the more complex magic-modules functionality like update_url and update_verb on the field itself? See https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/compute/Address.yaml#L201 as an example

Reply:
Done.